manifests/golang.yml

```diff
@@ -355,9 +355,9 @@ tests/:
       Test_TelemetryInstallSignature: missing_feature
     test_trace_sampling.py:
       Test_Trace_Sampling_Basic: v1.37.0 # TODO what is the earliest version?
-      Test_Trace_Sampling_Globs: missing_feature
-      Test_Trace_Sampling_Resource: missing_feature
-      Test_Trace_Sampling_Tags: missing_feature
+      Test_Trace_Sampling_Globs: v1.60.0-dev
```

Comment:
only for double check....Do we want to set to "1.60.0-dev"?
1.60.0 > 1.60.0-dev

Reply:
Current dev version: 1.61.0-dev
Current prod version: 1.60.0

Reply:
I will set it to 1.60.0 then